packages/lodestar-cli/package.json

```diff
@@ -23,7 +23,7 @@
     "lint:fix": "eslint --color --ext .ts src/ --fix",
     "pretest": "yarn run check-types",
     "test:unit": "TS_NODE_PROJECT=tsconfig.test.json nyc --cache-dir .nyc_output/.cache -e .ts mocha --file ./test/setup.ts --colors -r ts-node/register 'test/unit/**/*.test.ts'",
-    "test:e2e": "TS_NODE_PROJECT=tsconfig.test.json mocha --file ./test/setup.ts --colors --timeout 5000 -r ts-node/register 'test/e2e/**/*.test.ts'",
+    "test:e2e": "TS_NODE_PROJECT=tsconfig.test.json mocha --file ./test/setup.ts --colors --timeout 5000 --exit -r ts-node/register 'test/e2e/**/*.test.ts'",
```

Comment:
I failed to figure out why e2e does not stop, that's something related to libp2p because if I comment it out, it works.
Probably we leave it like this and resolve it after we migrate to next libp2p?

Reply:
libp2p should already be updated to the version that awaits discovery starting on libp2p start.
Not sure why its not exiting